elephant/statistics.py

```diff
@@ -629,3 +647,220 @@ def time_histogram(spiketrains, binsize, t_start=None, t_stop=None,
                                  sampling_period=binsize, units=bin_hist.units,
                                  t_start=t_start)
 
+
+"""Kernel Bandwidth Optimization.
+Copyright (c) 2014 Subhasis Ray
```

Comment:
You can't include GPL-licenced code in a BSD-licenced project. Can you ask Subhasis Ray if he is willing to relicence the code? (If you can't find his e-mail address, let me know)